### ISSUE TITLE
Summary export permission denied

### ISSUE THREAD

**Issue by edno** [grafana/k6-action#1]
When passing the flag `--summary-export results.json` then an error is raised:
```
level=error msg="failed to create summary export file" error="open results.json: permission denied"
```

The result is that no summary file is created.

Example of step:
```yaml
uses: k6io/action@v0.1
with:
  filename: scenarios/${{ github.event.inputs.scenario }}.js
  flags: --summary-export ${{ github.event.inputs.scenario }}.results.json
```

The expected result is that the flag works like on a local environment, and the `result.json` file is created and can further be exported using `actions/download-artifact@v2`
```yaml
uses: actions/upload-artifact@v2
with:
  name: ${{ github.event.inputs.scenario }}.results.json
  path: ${{ github.event.inputs.scenario }}.results.json
```

**Comment by edno** [comment]
A possible root cause is that the k6 docker image set the `USER` in the [Dockerfile](https://github.com/loadimpact/k6/blob/master/Dockerfile#L12), what is not recommended as per [Github Action documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#docker-container-filesystem).

**Comment by simonaronsson** [comment]
Hey,

Thank you for your report! I'll have a look at this. Contrary to GitHub's own recommendation, it's generally a bad practice to run anything in docker as root. I'll get back to you once I have more details.

Best,
Simme

**Comment by edno** [comment]
Happy New Year @simskij 

Any update on this issue?

If you have any suggestion for a workaround, I'll be happy to give it a try.

**Comment by simonaronsson** [comment]
Hi @edno,

This should be fixed as of `v0.2.0`, which I just released. 

Best,
Simme

**Comment by edno** [comment]
Confirmed. It works as expected 😃
